### ISSUE TITLE
kubectl taint says "tainted" when you remove a taint

### ISSUE THREAD

**Issue by davidopp** [kubernetes/kubernetes#43349]
This is obviously confusing:

```
$ kubectl taint nodes kubernetes-minion-group-tqsn x:NoExecute-
node "kubernetes-minion-group-tqsn" tainted
```

It should say "untained" or "taint removed" or something like that.

cc/ @kubernetes/sig-scheduling-bugs

**Comment by davidopp** [comment]
Would be nice to fix this in 1.6 but it's not a regression so we could fix it in 1.6.1

**Comment by liggitt** [comment]
Dupe of #43167
